fusionauth-netcore-client-test/fusionauth-netcore-client-test/test/io/fusionauth/FusionAuthClientTest.cs

```diff
@@ -295,11 +295,12 @@ public class FusionAuthClientTest {
 
       var response = test.client.RetrieveRefreshTokens((Guid) test.user.id);
       test.assertSuccess(response);
-      Assert.IsNull(response.successResponse.refreshTokens);
+      Assert.AreEqual(0, response.successResponse.refreshTokens.Count);
```

Comment:
Makes sense to me. The `refreshTokens` property is non-nullable, so a count on the list is a more accurate test.